distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestNsili.java

```diff
@@ -66,6 +64,10 @@
 
     private Thread mockServerThread;
 
+    private static final String CORBA_DEFAULT_PORT_PROPERTY = "org.codice.alliance.corba_default_port";
```

Comment:
The ORB is now shared with the Source and the Endpoint, so we don't want to package this too close to just the NSILI endpoint.